contracts/vaults/RibbonThetaVault.sol

```diff
@@ -491,8 +553,6 @@ contract RibbonThetaVault is DSMath, OptionsVaultStorage {
             shortAmount
         );
 
-        startAuction();
-
```

Comment:
you think we should start the auction in a separate tx?

Reply:
Removed this by accident, will add back